HLTrigger/Configuration/test/getEventContent.py

```diff
@@ -1,35 +1,58 @@
-#! /usr/bin/env python3
```

Comment:
`getEventContent.py`:
 - added `argparse` and support for `--dbproxy` option
 - minimised number of ConfDB queries (from 4 to 1)
 - output unchanged, but now the scripts outputs to stdout (not to a file whose name is hard-coded inside the script)